core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -5936,8 +5937,39 @@ enum DefaultConstants NADefaults::validateAndInsert(const char *attrName,
      }
      break;
 
-      default:  break;
-      }
+     // max char col length is 10M (10485760).
```

Comment:
this is an obsolete comment. We used to support upto 10M but now increased it to 16M.
will fix it for a later checkin.